user_guides/jakartaee/src/main/jbake/content/intro.adoc

```diff
@@ -459,38 +504,39 @@ Linux software that meet the following software requirements:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In addition to the instructions and requirements described in this
-document, all Jakarta EE 10 and Jakarta EE 10 Web Profile implementations must
+document, all Jakarta EE 10 Platform implementations must
 also pass the standalone TCKs for the following technologies:
 
-* Jakarta Batch 2.1
-* Jakarta Contexts and Dependency Injection 4.0
-* Jakarta Dependency Injection 2.0
-* Jakarta Bean Validation 3.0
-* Jakarta XML Binding 3.0
-* Jakarta JSON Binding 3.0
-* Jakarta JSON Processing 2.1
-* Jakarta RESTful Web Services 3.1
-
-For more information about the Jakarta Batch technology,
-see the specification at `https://jakarta.ee/specifications/batch/2.1/`
-
-For more information about the Jakarta Contexts and Dependency Injection technology, 
-see the specification at `https://jakarta.ee/specifications/cdi/4.0/`
-
-For more information about the Jakarta Dependency Injection,
-see the specification at `https://jakarta.ee/specifications/dependency-injection/2.0/` 
-
-For more information about the Jakarta Bean Validation technology, see the
-specification at `https://jakarta.ee/specifications/bean-validation/3.0/`
-
-For more information about the Jakarta XML Binding technology, see the
-specification at `https://jakarta.ee/specifications/xml-binding/3.0/`
-
-For more information about the Jakarta JSON Binding technology, see the specification at `https://jakarta.ee/specifications/jsonb/3.0/`
-
-For more information about the Jakarta JSON Processing technology, see the specification at `https://jakarta.ee/specifications/jsonp/2.1/`
+* Jakarta Activation -- see https://jakarta.ee/specifications/activation/ for additional details
```

Comment:
I looked at some previous EE CCR results and do not see Jakarta Activation TCK mentioned (e.g. [GlassFish 6.1](https://github.com/eclipse-ee4j/jakartaee-platform/issues/334)).

Reply:
What is driving the proposed transition from the Platform TCK to the individual TCKs for both Activation and Mail?  Are the existing tests in the Platform TCK insufficient in some way even though they've always constituted compliance in the past?  Are these proposed requirements intended to be additive to what is in the Platform TCK or a replacement for the tests that are there?

Reply:
New tests have been developed and are not migrated back to the Platform TCK. Therefore, while the Platform TCK runs Mail tests, it is not complete. Activation has always been a requirement as well -- in previous versions, the Platform TCK verified compatibility (and also for Mail). Now that the TCK has been migrated out of the Platform TCK, the external TCK must be run.